src/main/java/graphql/ExecutionResult.java

```diff
@@ -11,6 +11,13 @@
 
     List<GraphQLError> getErrors();
 
-    Map<Object,Object> getExtensions();
+    Map<Object, Object> getExtensions();
```

Comment:
Well, I use extension in my work, so I prefer to leave it as it is.
Also, I think it is better not to delete it and maybe do a //TODO.
About the immutable, it is better to do it immutable after graphql-java supports extension, that way people that do use extensions (like me) don't need to make another instance.

But even if you think I'm wrong, I think it's for another PR, don't you?